pulp_smash/config.py

```diff
@@ -131,6 +133,14 @@ def _get_pulp_3_content_role():
                 "selinux enabled": {"type": "boolean"},
             },
         },
+        "general": {
+            "additionalProperties": False,
+            "required": ["timeout"],
```

Comment:
I think timeout is optional, if user does not want to provide it, then pulp-smash uses the default value.